### ISSUE TITLE
high level terms related to ophthalmology

### ISSUE THREAD

**Issue by psergouniotis** [obophenotype/human-phenotype-ontology#3358]
Some of the high level terms relating to ophthalmology need revision.

For example: 
* "Abnormality of the eye" is a subclass of "Phenotypic abnormality"; it should perhaps be a subclass of "Abnormality of head" (see comment [A] below)
* Some subclasses of "Abnormality of the eye" such as optic nerve misrouting are not strictly speaking eye pathologies. It would be reasonable to introduce a superclass called "Abnormality of the visual system" (see comment [A] below).
* There is "Abnormal morphology of the ocular adnexa" but no "Abnormality of the ocular adnexa" (see comment [C1] below).
* It is unclear what is the difference between "Abnormal eye morphology" and "Abnormal morphology of the globe".

I recommend that we gradually have a look at this but the main issue is where should "abnormality of the eye" be (I guess we need to stay consistent with uberon) and whether "abnormality of the visual system" should be introduced. 

(continued)

**Comment by psergouniotis** [comment]
Please consider the following:

[A]
- Abnormality of the eye"  (presently a subclass of "Phenotypic abnormality") to be a subclass of "Abnormality of the head".
- To maintain consistency "Abnormality of the ear" (presently a subclass of "Phenotypic abnormality") should also be a subclass of "Abnormality of the head".

Additionally consider introducing "Abnormality of the sensory nervous system" as a subclass of "Abnormality of the nervous system" (I note that this also needs revision). "Abnormality of the eye" and "Abnormality of the ear" could be potentially subclasses of "Abnormality of the sensory nervous system". 

I would favour the above but, alternatively,  one could consider adding "Abnormality of sensory organ" as a subclass of "Phenotypic abnormality", "Abnormality of the eye" and "Abnormality of the ear" could be potentially subclasses of "Abnormality of sensory organ".

**Comment by psergouniotis** [comment]
[B]
I think that "**Abnormality of the visual system**" would be a reasonable term to add. If you would be OK with that I would recommend having it as a superclass of "Abnormality of the eye".

"Abnormality of the visual system" could be a subclass of "Abnormality of the head" and Abnormality of the sensory nervous system".

**Comment by psergouniotis** [comment]
[C]
Two other terms related to ophthalmology are:
- Abnormality of the orbital region
- Abnormality of the periorbital region
These are subclasses of "Abnormality of the face"

[C1]  In my mind, "Abnormality of the orbital region" refers to the bony orbit and ocular adnexa, not the globe.
Therefore "Abnormality of globe size", "Abnormality of globe location" and "Coloboma" should NOT be subclasses here. 

The subclasses of  "Abnormality of the orbital region"  should therefore be:
- **Abnormality of the ocular adnexa** (new term; the current term "Abnormal morphology of the ocular adnexa" should be its subclass)
- Abnormality of bony orbit of skull (although technically this is no face)
- Hamartoma of the orbital region.
- Orbital cyst

[C2]  "Abnormality of the periorbital region".
The following should NOT be subclasses of "Abnormality of the periorbital region": 
"Orbital cyst", "Abnormality of bony orbit of skull".

"Abnormality of the eyebrow" should take the place of "Abnormal eyebrow morphology" in the hierarchy. "Abnormal eyebrow morphology" should be a subclass of "Abnormality of the eyebrow".

**Comment by psergouniotis** [comment]
I would really appreciate any thoughts. Once we sort these out we can move on to tidying up the 3 main classes: "Abnormality of the visual system", "Abnormality of the eye" and "Abnormality of the ocular adnexa".  

A key question would be at which level would you like to split morphology from physiology.

**Comment by pnrobinson** [comment]
Currently we split morphology and physiology at a very high level (right under Eye, for instance). 
With the very first version of the HPO, we did not distinguish between morphology and pheysiology in this way, and there are still some remnants of this in the HPO. The vast majority of the "Abnormality of XYZ" terms should actually be Abnormal XYZ morphology -- or -- Abnormal XYZ physiology. We will then have Abnormality of ORGAN only for the top level-- if this makes sense.

**Comment by pnrobinson** [comment]
@psergouniotis But Abnormality of the visual system would not cover things like lacrimation and ocular adnexa --  I am not sure this is the optimal way of doing things?

**Comment by psergouniotis** [comment]
Thanks for clarifying the morphology/physiology issue Peter. 
In terms of ophthalmology, having "Abnormality of the eye" and, as subclasses, "Abnormal eye morphology" and 'Abnormal eye physiology" makes sense.  
I am mainly raising this regarding ocular adnexa and abnormalities relating to the optic nerve or optic chiasm which are technically not eye. We could discuss these in a different thread and further to guidance from the ERN-EYE neuroophthalmology workgroup. 

For the time being, would it be possible to consider introducing "Abnormality of the ocular adnexa" and making he current term "Abnormal morphology of the ocular adnexa" as its subclass? 

Also, would it be possible to move around the terms discussed in [C1 & C2] above?

**Comment by psergouniotis** [comment]
Further to your second comment. Indeed, "Abnormality of the visual system" would not cover ocular adnexa but "Abnormality of the eye" does not cover them either ("Abnormality of the ocular adnexa" should be a subclass of "Abnormality of the orbital region".

Technically increased lacrimation is coming from the lactimal apparatus and should be under ocular adnexa (and not eye as it is at present). I think we should make a clear distinction as suggested above and then move terms accordingly.

What "Abnormality of the visual system" would solve is some neuro-ophthalmic phenotypes relating to the chiasm, optic radiation etc.

**Comment by psergouniotis** [comment]
To clarify further I think the following hierarchy would work well for ophthalmology terms.


Abnormality of the head
  | Abnormality of the visual system
  |   |  Abnormality of the eye (or abnormality of the globe)
  |   |  Abnormality of the optic nerve
  |  Abnormality of the face
  |   | Abnormality of the periorbital region
  |   | Abnormality of the orbital region
  |   |   |  Abnormality of the ocular adnexa
  |   |   |   | Abnormality of the nasolacrimal system
  |   |   |   | Abnormality of the conjunctiva
  |   |   |   | Abnormality of the eyelid
  |   |   |   | Abnormality of the eyebrow
  |   |   |   | Abnormality of the extraocular muscles (this is perhaps questionable - should it be under eye?)
  |   |   |   | Abnormality of the common tendinous ring


Also 
Abnormality of the nervous system
  | Abnormality of the sensory nervous system
  |   | Abnormality of the visual system

**Comment by cmungall** [comment]
The challenge with the above scheme is that you end up with things like cataracts being inferred to be nervous system abnormalities (lens is definitely part of the eye globe, which in the proposed scheme is part of the visual system and from there to nervous system).

This is something we've wrestled with systems in general with uberon, when you examine what are commonly taken to be their parts, the relationship is in fact overlaps. So you could have "[abnormality of] nervous system portion of eye" for example, but it's a bit odd.

We have also considered a relationship weaker than part-of but stronger than overlaps indicating that the function of the organ is realized by the system in question, but it's not clear how that should propagate to HPO. E.g. "Abnormality of the visual system" could be reconceived as "Abnormality of the functional parts of the visual system" but that's pretty odd.

TL;DR - I think where systems are added the placement of organs under them should be conservative, and leave uberon inference to place the organ parts their (e.g. under Ab-visual system we'd have the retina and its neural projections)

**Comment by psergouniotis** [comment]
Many thanks for the response.  Fair point about using systems.

I guess we are then left with this:
Abnormality of the head
  | Abnormality of the eye (or globe)
  |  Abnormality of the face **
  |   | Abnormality of the periorbital region **
  |   | Abnormality of the orbital region **
  |   |   | Abnormality of the ocular adnexa
  |   |   |   | Abnormality of the nasolacrimal system
  |   |   |   | Abnormality of the conjunctiva
  |   |   |   | Abnormality of the eyelid
  |   |   |   | Abnormality of the eyebrow
  |   |   |   | Abnormality of the extraocular muscles **
  |   |   |   | Abnormality of the common tendinous ring **


The issue with this is what do we do with optic nerve terms. The term
 | Abnormality of the optic nerve 
is presently under "Abnormality of the fundus".  
It would be more accurate to have it under "Abnormality of the cranial nerves" (should be perhaps renamed to "Cranial nerve abnormality by the way)

Notably, it subclass 
 |  Abnormality of the optic disc 
should be a subclass of "Abnormality of the fundus". 


** What would be your view for
  |  Abnormality of the extraocular muscles and
  |  Abnormality of the common tendinous ring?

The are technically in the orbital region but not in the face (I would think). Should 
  |   | Abnormality of the periorbital region
  |   | Abnormality of the orbital region
be subclasses of "Abnormality of the head" rather than "Abnormality of the face"?

**Comment by pnrobinson** [comment]
@psergouniotis I have moved some of the hierarchy as discussed without making major changes. I would suggest for now that it is better to wait with these changes until perhaps we can have a follow-up workshop or at least a small meeting. It is difficult to get absolutely everything exactly right without causing other problems, as Chris mentions above. I will close this issue now as I have addressed many of the issues.

**Comment by psergouniotis** [comment]
OK. Thanks for making some of the changes. Should we perhaps leave this open?
